from0k2bp.tex

```diff
@@ -69,32 +69,31 @@
 might find this investigation at least partly as interesting to read as
 I found it to construct!
 
-\textbf{Knowledge prerequisites}: you're not assumed to know anything
-about elliptic curves, except that points add, can be expressed as
-scalar multiples, i.e. the standard public-private key relationship like
-$P=kG$, with $x$ the scalar value which is the private key, and understand why it
+\textbf{Knowledge prerequisites}: You are not assumed to know anything
+about elliptic curves (EC) except that EC points can be added and also EC points 
+can be expressed as scalar multiples, i.e. the standard public-private key relationship like
+$P=kG$ where $k$ is a scalar value which represents the private key, and understand why it
 makes sense to say something like $aP = axG$, and that if $P-Q=H$ and $Q=rsG$ then the private key
 of $H$ would be $(x-rs)$. No deeper mathematics about the \emph{construction} of
 elliptic curves will be relevant here. If you've already taken the time
 to understand e.g. ECDSA signatures or Schnorr signatures, you'll find
 nothing here to trouble you.
 
-Similar comments apply to hash functions (which play only a very minor
-role in most of this document).
+Similar comments apply to hash functions which play only a very minor
+role in most of this document.
 
-\emph{Very} basic linear algebra (let's say: how to solve a set of
-simultaneous linear equations; what a matrix inverse is) will be very
+\emph{Very} basic linear algebra\footnote{Let's say: How to solve a set of
```

Comment:
Not sure, don't think I'd want to overdo it with footnotes. Obviously debatable, but I feel they should be reserved for more substantive points than this.

Reply:
Updated comment: on further reading, I think *most* of the things you changed to footnotes work quite well as footnotes. So I'm not inclined to ask you to change them.

Reply:
There are pros and cons for footnotes and for parentheses. My issue with parentheses is that it breaks my flow of reading. So I find texts with footnotes more convenient. But that's just my personal taste. Not sure what others like better.